### PR TITLE
updates from manifestor that resolves #412, resolves #376, and closes…

### LINKED ISSUE #376: Make overlays fade out when in detail perspective, fade in when in overview perspective.

### LINKED ISSUE #412: Fix weird layout change stuff with frame and label

### PR DESCRIPTION
**Pull request by aeschylus**
… #119

### CODE REVIEW

**Comment by mejackreed** [comment]
I'm noticing some strange behavior here:

It seems as if "paged" is default based on the viewing hint, is this correct? I'm noticing some issues with bc421tk1152

This seems to be causing some state issues and the viewer is not behaving appropriately. Can we either, fix this in manifestor or update the viewer to handle this correctly?

File: app/assets/javascripts/vendor/manifestor.js [hunk 1]
```diff
@@ -3181,6 +3188,15 @@ var manifestor = function(options) {
     canvasImageStates(canvasStates);
   }
 
+  function addMainImageObj(id, osdTileObj) {
+    var canvasStates = canvasImageStates();
+    console.log('added main image');
```
Comment (mejackreed) [inline app/assets/javascripts/vendor/manifestor.js]: Can this be removed?

File: app/assets/javascripts/vendor/manifestor.js [hunk 2]
```diff
@@ -2912,16 +2899,25 @@ var manifestor = function(options) {
...
-      dummyObj.setPosition(new OpenSeadragon.Point(xi(t), yi(t)), true);
-      dummyObj.setWidth(d.canvas.width, true);
-      dummyObj.setHeight(d.canvas.height, true);
+      if (dummyObj) {
+        dummyObj.setPosition(new OpenSeadragon.Point(xi(t), yi(t)), true);
+        dummyObj.setWidth(d.canvas.width, true);
+        dummyObj.setHeight(d.canvas.height, true);
+      }
+      if (mainImageObj) {
+        console.log('yes mulowd');
```
Comment (mejackreed) [inline app/assets/javascripts/vendor/manifestor.js]: Can this be removed?

**Comment by aeschylus** [comment]
Yeah, I can fix these up. Sorry.

**Comment by aeschylus** [comment]
I may not be checking the paged viewing hint correctly as well.

**Pull request closed without merging**